vllm/entrypoints/chat_utils.py

```diff
@@ -440,6 +440,13 @@ def _parse_chat_message_content_parts(
         if mm_placeholder_counts:
             text_prompt = _get_full_multimodal_text_prompt(
                 mm_placeholder_counts, text_prompt)
+        if chat_template_text_format == "openai":
+            role_content = [
+                ChatCompletionContentPartTextParam(type="text",
+                                                   text=text_prompt)
+            ]
+            return [ConversationMessage(role=role,
+                                        content=role_content)]  # type: ignore
```

Comment:
Let's update the type definition in `ConversationMessage` so we don't have to ignore this.

Reply:
@DarkLight1337 Removing that type ignore will trigger a lot of errors in places that are not relevant to this PR. Do you still want me to fix those or should it be a separate PR?